hail/src/main/scala/is/hail/utils/richUtils/RichVal.scala

```diff
@@ -0,0 +1,6 @@
+package is.hail.utils.richUtils
+
+case class RichVal[A](a: A) extends AnyVal {
+  def &[B](f: A => B): B =
```

Comment:
I'm not sure about this. We use `&` other places, like on `CodeBoolean` and `CodeInt`, that could clash with this. Where are you using this, and could it be a regular method instead of an operator?

Reply:
aaaand it's gone.